usaspending_api/search/migrations/0008_awardsearch_table.py

```diff
@@ -115,7 +119,7 @@ class Migration(migrations.Migration):
         # Trick Django into believing this is a foreign primary key for purposes of using the ORM,
         # but avoid the headache that comes with foreign keys and the primary key constraint
         migrations.RunSQL(
-            sql="ALTER TABLE rpt.award_search DROP COLUMN id",
+            sql=";".join(additional_sql),
```

Comment:
If we aren't using that SQL anywhere else I would say put it here instead of splitting it between two places.